### PR TITLE
[Dy2stat] Add Ptb Lm Test for V2 APIs

### PR DESCRIPTION
**Pull request by zhhsplendid**
### PR types

Others
### PR changes

Others
### Describe

Add Ptb lm Test for V2 APIs.

We used tool from https://www.paddlepaddle.org.cn/documentation/docs/zh/2.0-beta/guides/migration_cn.html#paddle1-xpaddle2-0beta to do v1.x to v2 transformation.

This PR pulled changes from #27430, please DO NOT merge before #27430 is merged

### CODE REVIEW

**Review by Aurelius84 — DISMISSED** [review]
LGTM

**Review by liym27 — DISMISSED** [review]
LGTM.

---

Mark paddle.fluid.layers.reshape -> paddle.reshape

**Comment by zhhsplendid** [comment]
> LGTM.
> 
> Mark paddle.fluid.layers.reshape -> paddle.reshape

Not only that, I found more and I'm going to change

**Comment by zhhsplendid** [comment]
Refine all fluid APIs except Embedding which is different to 2.0 Embedding

**Review by Aurelius84 — APPROVED** [review]

File: python/paddle/fluid/tests/unittests/dygraph_to_static/test_ptb_lm_v2.py [hunk 1]
```diff
@@ -0,0 +1,314 @@
...
+        self.num_layers = num_layers
+        self.num_steps = num_steps
+        self.dropout = dropout
+        self.simple_lstm_rnn = SimpleLSTMRNN(
+            hidden_size,
+            num_steps,
+            num_layers=num_layers,
+            init_scale=init_scale,
+            dropout=dropout)
+        self.embedding = paddle.fluid.dygraph.nn.Embedding(
```
Comment (liym27) [inline python/paddle/fluid/tests/unittests/dygraph_to_static/test_ptb_lm_v2.py:139]: paddle.fluid.dygraph.nn.Embedding -> paddle.nn.Embedding

**Review by liym27 — APPROVED** [review]
LGTM

**Pull request merged**